pkgs/servers/persistent-evdev/persistent-evdev-udev-rules.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "persistent-evdev-udev-rules";
+  version = "unstable-2020-07-15";
+
+  udevRules = fetchurl {
+    url = "https://raw.githubusercontent.com/aiberia/persistent-evdev/bc38d5b6213b984d6b184f96a8ea4d9bef792f53/udev/60-persistent-input-uinput.rules";
+    sha256 = "19lrlgydpyggk4w14966v1yfxrkizjhqmfzxrpilwx949ij83v8i";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
```

Comment:
fetchurl supports both `postFetch` to run this shell fragment and `meta` to add metadata. no need to wrap it in another derivation only to add those two. (only snag is that fetchurl *doesn't* have a pname, only a `name`)